ibm_catalog.json

```diff
@@ -0,0 +1,193 @@
+{
+    "products": [
+        {
+            "name": "deploy-arch-ibm-slz-c2s-vpn",
+            "label": "Client to Site VPN access to VPC landing zone",
+            "product_kind": "solution",
+            "tags": [
+                "network",
+                "ibm_created",
+                "target_terraform",
+                "terraform",
+                "reference_architecture",
+                "solution"
+            ],
+            "keywords": [
+                "vpc",
+                "slz",
+                "IaC",
+                "infrastructure as code",
+                "terraform",
+                "solution",
+                "VPN",
+                "OpenVPN"
+            ],
+            "short_description": "Creates client-to-site VPN connectivity to landing zones using VPN Server ",
+            "long_description": "By default, the landing zone patterns are configured with private networks not available over the internet.  To access these networks, there are several connectivity options.  This deployable architecture pattern configures the client-to-site VPN Server connectivity with only a few required inputs to configure it within an existing landing zone.  Once deployed, you can install an OpenVPN client application and import a profile from the VPN Server on the devices you want to access the VPN.  The configuration can include a list of users that will be provided access to the private network, controlled by IBM Cloud IAM.\n",
+            "offering_docs_url": "https://ibm.ent.box.com/notes/1212695139323",
```

Comment:
looks like private box note here

Reply:
It's ok for now - this will be replaced by the link to doc once it is published.